src/views/poi_bloc/Information.jsx

```diff
@@ -0,0 +1,103 @@
+/* global _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import renderStaticReact from 'src/libs/renderStaticReact';
+import AccessibilityBlock from '../../views/poi_bloc/Accessibility';
+import BreweryBlock from '../../views/poi_bloc/Brewery';
+import InternetAccessBlock from '../../views/poi_bloc/InternetAccess';
+import WikiBlock from '../../views/poi_bloc/Wiki';
+
+function findBlock(blocks, toFind) {
+  for (let i = 0; i < blocks.length; ++i) {
+    if (blocks[i].type === toFind) {
+      return blocks[i];
+    } else if (blocks[i].blocks !== undefined) {
+      const ret = findBlock(blocks[i].blocks, toFind);
+      if (ret !== null) {
+        return ret;
+      }
+    }
+  }
+  return null;
+}
+
+export default class InformationBlock extends React.Component {
+  static propTypes = {
+    block: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { isCollapsed: true };
+
+    this.accessibilityBlock = findBlock(this.props.block.blocks, 'accessibility');
+    this.breweryBlock = findBlock(this.props.block.blocks, 'brewery');
+    this.internetAccessBlock = findBlock(this.props.block.blocks, 'internet_access');
+    this.wikiBlock = this.props.block.blocks.find(b => b.type === 'wikipedia');
+
+    this.expandCollapse = this.expandCollapse.bind(this);
+  }
+
+  expandCollapse() {
+    this.setState(state => ({
+      isCollapsed: !state.isCollapsed,
+    }));
+  }
+
+  checkSubBlocks() {
+    return this.accessibilityBlock || this.breweryBlock || this.internetAccessBlock;
+  }
+
+  getText() {
+    if (!this.state.isCollapsed) {
+      return _('Services & information');
+    }
+    const elems = [];
+    if (this.accessibilityBlock) {
+      elems.push(renderStaticReact(
```

Comment:
I had an issue because of my `array.join(' - ')` because I fixed it by inserting by hand the text. Looks a bit less nice but at least, no more static rendering!